src/main.rs

```diff
@@ -17,6 +17,7 @@ use std::{
 mod config;
 mod utils;
 
+const VERSION: &str = "v1.0.0";
```

Comment:
Instead of keeping track of the version in two different places, we should instead use the version in `Cargo.toml`

For example,
```rust
let version = option_env!("CARGO_PKG_VERSION");
```